grpc/server/server_test.go

```diff
@@ -0,0 +1,55 @@
+package server
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+	commonv1 "github.com/nanzhong/tstr/api/common/v1"
+	"github.com/nanzhong/tstr/grpc/types"
+)
+
+type accessTokenBuilder struct {
+	accessToken *commonv1.AccessToken
+}
+
+func NewAccessTokenBuilder() *accessTokenBuilder {
```

Comment:
I think this function should be non-exported since it shouldn't be used outside of this package.

Reply:
ack!